specification/cloudshell/resource-manager/Microsoft.Portal/stable/2018-10-01/CloudShell.json

```diff
@@ -0,0 +1,952 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-04-01-preview",
+    "title": "CloudShellClient"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "Impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/Microsoft.Portal/locations/{location}/userSettings/{userSettingsName}": {
```

Comment:
There are no  list/collection APIs in this RP (e.g. `/providers/Microsoft.Portal/locations/{location}/userSettings/`), I assume that is because of the nature of this RP and has a prior exception.